worker/pyproject.toml

```diff
@@ -29,6 +29,13 @@ deploy = [
     "gunicorn>=21.2.0",
 ]
 
+dev = [
```

Comment:
@ritwik-g is there any reason why don't include tox here? Is it not ideal to run it locally?

Reply:
First of all it needs to be added in to the pyproject of the root. Because tox is maintained there. But I ran in to some dependency conflicts due to which I couldn't add it there.